packages/dcos-log/build

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+export GOPATH=$GOPATH:/pkg
+mkdir -p /pkg/src/github.com/dcos
+# Create the GOPATH for the go tool to work properly
+mv /pkg/src/dcos-log /pkg/src/github.com/dcos/
+cd /pkg/src/github.com/dcos/dcos-log
+
+make install
+# Copy the build from the bin to the correct place
+cp -r /pkg/bin/ "$PKG_PATH"
+
+master_service=${PKG_PATH}/dcos.target.wants_master/dcos-log.service
```

Comment:
the same reason as i mentioned in first comment. To be consistent with DC/OS EE. Unfortunately we need different service accounts for the same service running on master and agent nodes :( I could update the build script but it would mean we'd have to diverge the EE build script.